code/modules/reagents/reagent_dispenser.dm

```diff
@@ -86,10 +86,7 @@
 
 	// We do this here to avoid putting the vessel straight into storage.
 	// This is usually handled by afterattack on /chems.
-	// The item must be an open container, but food items should not be filled from sources like this.
-	// They're open in order to add condiments, not to be poured into/out of.
-	// TODO: Rewrite open-container-ness or food to make this unnecessary!
-	if(storage && ATOM_IS_OPEN_CONTAINER(W) && !istype(W, /obj/item/food) && user.a_intent == I_HELP)
+	if(storage && ATOM_IS_OPEN_CONTAINER(W) && user.a_intent == I_HELP)
```

Comment:
presently i think that's just `ATOM_IS_OPEN_CONTAINER(X)` but it could be nice to make it more contextual i suppose. ~~unsure if it fits in this PR bc there's not really much use for it atm~~ you can pour pills into fire_sources because they don't check `ATOM_IS_OPEN_CONTAINER(W)` oops. yeah there's a good use-case for this

Reply:
it's checked in `standard_pour_into()`, i wasn't sure